book/06-github/sections/2-contributing.asc

```diff
@@ -457,7 +457,7 @@ When rendered, it would look something like <<_md_emoji>>.
 .Heavy emoji commenting.
 image::images/markdown-07-emoji.png[Emoji]
 
-Not that this is incredibly useful, but it does add an element of fun and emotion to a medium that is otherwise hard to convey emotion in.
```

Comment:
I am pretty sure this version was correct.

Reply:
Ha! You’re completely right. Want to submit a pr that fixes it?

Reply:
Done (#1268).